packages/devtools_app/lib/src/framework/framework_core.dart

```diff
@@ -93,6 +95,8 @@ abstract class FrameworkCore {
 
   static bool vmServiceInitializationInProgress = false;
 
+  static EditorThemeManager? themeManager;
```

Comment:
make this private

Reply:
This var is getting set in `_initDTDConnection` which is a separate function, so I think it has to be accessible?

Reply:
It should still be accessible since it is private within the same class where _initDTDconnection is defined